samples/bluetooth/broadcaster_multiple/README.rst

```diff
@@ -0,0 +1,23 @@
+.. _bluetooth-broadcaster-multiple-sample:
+
+Bluetooth: Broadcaster Multiple
```

Comment:
I know it's "Broadcaster Multiple (advertising sets)" but I think "Multiple Broadcaster" reads better. what do you think